code/obj/window.dm

```diff
@@ -1046,11 +1046,11 @@
 // flock windows
 
 /obj/window/auto/feather
-	default_material = "gnesisglass"
 
 /obj/window/auto/feather/New()
 	connects_to += /turf/simulated/wall/auto/feather
 	..()
+	setMaterial(getMaterial("gnesisglass"), appearance = FALSE, setname = FALSE)
```

Comment:
Though these windows were supposed to inherit from the regular windows?

Reply:
they do?

Reply:
```suggestion
```
Err yeah, I don't think this is needed at all, testing it seems to already have the gnesisglass material from window/feather()